src/js/components/aboutTheData/dataMapping/timeFilters.js

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
This should be in the dataMapping folder.

Reply:
Im following the precedent set by `components/aboutTheDAta/dataMapping/modals`